dashboard/test/controllers/sessions_controller_test.rb

```diff
@@ -352,4 +321,92 @@ class LtiAccountLinkingSignInPageTest < ActionDispatch::IntegrationTest
       assert_template partial: 'devise/sessions/_login'
     end
   end
+
+  describe 'GET lockout' do
+    let(:user) {create(:parent_managed_student)}
+
+    let(:user_is_locked_out) {true}
+    let(:latest_permission_request) {nil}
+
+    around do |test|
+      Timecop.freeze {test.call}
+    end
+
+    before do
+      Policies::ChildAccount.stubs(:locked_out?).with(user).returns(user_is_locked_out)
+      Queries::ChildAccount.stubs(:latest_permission_request).with(user).returns(latest_permission_request)
+
+      sign_in user
+    end
+
+    it 'assigns @pending_email with empty string' do
+      get :lockout
+      _(assigns[:pending_email]).must_equal ''
+    end
+
+    it 'assigns @disallowed_email with empty string' do
+      get :lockout
+      _(assigns[:disallowed_email]).must_equal ''
```

Comment:
I think you'll be the first to actually use minitest expectations and minitest/spec in the codebase in spite of it being an option for a long while. 😅

I'm happy with this.

Reply:
I know the `_` can be opinionated... I wonder if anybody has an opinion on `_(...)` vs the `expect(...)` alias, but I like the `_` better. Easier to scan for and I already easily know that it is representing an expectation and an assert. All good to me!